Readme.md

```diff
@@ -74,34 +124,53 @@ Cloning...Succeeded
 Fetching commits and trees from origin (no cache server)...Succeeded
 Configuring Watchman...Succeeded.
 Validating repo...Succeeded
+```
+
+Then, navigate into the repository's `src` directory.
```

Comment:
I'm wondering if we should make a bigger deal out of the fact that we created and extra intermediate directory.  Normal Git users will be expecting `ForTests` to be the root of the repo and contain the .git directory and etc.

Perhaps explain why work dir root as well as .git is down a level.
And maybe give a brief comment on what is in the actual `ForTests` directory besides `src`.

Reply:
I see you have a comment about this in the paragraph after the `ls`.  Maybe pull that up here and make a stronger statement here.  Just a thought.

Reply:
I should link to the question in the FAQ: https://github.com/microsoft/scalar/blob/main/docs/faq.md#why-does-scalar-clone-create-a-reposrc-folder